core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -40,6 +40,7 @@ oppia.controller('FeedbackTab', [
 
     $scope.activeThread = null;
     $scope.userIsLoggedIn = null;
+    $scope.updatedThreadFlag = 0;
```

Comment:
May be change the var name to -> `isThreadUpdated` since it's used as Boolean variable here. Also, assign value as either true/false for better readability.  Ditto everywhere else.

Reply:
Done